Libraries/Text/RCTTextField.m

```diff
@@ -279,4 +296,27 @@ - (BOOL)resignFirstResponder
   return result;
 }
 
+
++(NSMutableDictionary*)getAcessoryMapping {
```

Comment:
nit: call this just `accessoryMapping`. Objective-C assessors are not prefixed with get.